tests/onnxruntime/test_diffusion.py

```diff
@@ -82,12 +82,18 @@ def generate_prompts(batch_size=1):
     return inputs
 
 
+IMAGE = None
+
+
 def generate_images(height=128, width=128, batch_size=1, channel=3, input_type="pil"):
     if input_type == "pil":
-        image = load_image(
-            "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main"
-            "/in_paint/overture-creations-5sI6fQgYIuo.png"
-        ).resize((width, height))
+        global IMAGE
+        if IMAGE is None:
+            # Load a sample image from the Hugging Face Hub
+            IMAGE = load_image(
+                "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main/in_paint/overture-creations-5sI6fQgYIuo.png"
+            )
+        image = IMAGE.resize((width, height))
```

Comment:
to avoid load_image failing because of multiple calls, the method itself is very error prone when many calls are made to the same url in parallel.